api/ombpdf/management/commands/pdf_to_db.py

```diff
@@ -23,8 +25,13 @@ def handle(self, *args, **options):
         policy_name = options['policy'] or Path(fp.name).stem.upper()
         policy = fetch_policy(policy_name)
 
-        if policy is None:
-            raise CommandError(f"Policy '{policy_name}' not found.")
+        with reversion.create_revision():
+            if policy is None:
+                raise CommandError(f"Policy '{policy_name}' not found.")
+                policy.workflow_phase = WorkflowPhases.failed.name
```

Comment:
`policy` is `None` here -- won't this fail? Would it make sense to set the workflow phase to `failed` around original line 30 and `cleanup` at the end of the file? If an exception's thrown, we'd stay in the `failed` state.